src/config/Config.zig

```diff
@@ -1851,6 +1851,13 @@ keybind: Keybinds = .{},
 /// must always be able to move themselves into an isolated cgroup.
 @"linux-cgroup-hard-fail": bool = false,
 
+/// Start new windows maximized. This setting applies to new windows and
+/// does not apply to tabs, splits, etc. However, this setting will apply to all
+/// new windows, not just the first one.
+///
+/// This setting is overwritten by `fullscreen` option.
+@"gtk-maximize": bool = false,
```

Comment:
I don't think that we want to prefix this config with `gtk-` as this could be used on macOS as well.

Reply:
Sure, it makes more sense, I removed `gtk-` from the name.